web-ui/src/main/resources/catalog/views/default/less/gn_variables_default.less

```diff
@@ -14,6 +14,8 @@
 @navbar-default-link-color: @gray;
 @navbar-default-link-hover-color: @gray-dark;
 @text-muted: #707070;
+// width of modal dialog
+@modal-md: 700px;
```

Comment:
Are you sure this is needed? I mean, this will apply to all modals, so it may have side effects.

Reply:
It only effects the modal dialogs on tablet sized screens. I increased the value to be able to display the whole table, there simply wasn't enough room for table, it 'broke' out of the dialog.

Reply:
Ok, got it, this is fine then. Thanks